app/components/meeting-form.js

```diff
@@ -1,24 +1,42 @@
 import Component from '@glimmer/component';
 import { action } from "@ember/object";
 import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency-decorators';
 import { inject as service } from '@ember/service';
 
-export default class MeetingForm extends Component{
+export default class MeetingForm extends Component {
+
   @tracked aanwezigenBijStart;
   @tracked voorzitter;
   @tracked secretaris;
   @tracked zitting;
+  @tracked behandelingen;
+  @service store;
 
   constructor() {
     super(...arguments);
     this.zitting = this.args.zitting;
     this.secretaris = this.args.zitting.get('secretaris');
     this.voorzitter = this.args.zitting.get('voorzitter');
     this.aanwezigenBijStart = this.args.zitting.get('aanwezigenBijStart');
+    this.fetchBehandelingen.perform();
+
+  }
+
+  /** @type {import("ember-concurrency")} */
+  @task
+  fetchBehandelingen = function*() {
+    /** @type {import("../models/agendapunt").default[]} */
```

Comment:
importing the typedefs first like you do in app/components/treatment/participation.js makes this more readable.